### DISCUSSION TITLE
After the video has been corrupted, it goes out of sync.

### DISCUSSION THREAD

**Discussion by Tanggue** [aclap-dev/video-downloadhelper#1222] [Help and Q&A]
### Your question

After downloading and playing the video, the pixels suddenly become larger and an error occurs. The video plays but is out of sync. Is there a solution to this problem?
Typically, this phenomenon occurs more in large files with high bitrates or long video durations.

![Uploading 화면 캡처 2024-09-07 024025.png…]()

### Addon & CoApp version and details

version: 9.1.0.42
target: mozilla
channel: stable

**Comment by Mattmjs** [comment]
Your image doesn't work, you need to edit it out and add a proper image and share a video link where the problem occurs.
